client/me/purchases/cancel-purchase/get-plan-cancellation-features.jsx

```diff
@@ -0,0 +1,57 @@
+import { getPlan, isMonthly } from '@automattic/calypso-products';
```

Comment:
This file is simply a jsx version of client/me/purchases/pre-cancellation-dialog/get-plan-cancellation-features.tsx. We should either remove the tsx file here, or add a TODO note making it clear that we will remove the tsx version as a follow-up. Otherwise we risk making other developers confused about why there are two seemingly identical files.

Reply:
My idea was to remove the TSX file in PR to clean up the work done for pre-cancellation modal. Where should I add the TODO?

Reply:
The beginning of this file will be good enough :)

Reply:
done